### PR TITLE
Migrated Code of Conduct page to layer5-ng

### LINKED ISSUE #848: [NG] Migrate Code of conduct page to layer5-ng
**Current Behavior**
The current website has code of conduct over [here.](https://layer5.io/terms/code-of-conduct) This is to be migrated to layer5-ng. 

**Desired Situation**
Create a page for code of conduct in the pages folder.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by ashutosh1401**
**Description**

This PR fixes #848 and I have successfully migrated Code of Conduct page to layer5-ng in React. 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/pages/terms/code-of-conduct.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ThemeProvider } from "styled-components";
+import theme from "../../theme/blog/themeStyles";
+import { GlobalStyle } from "../../sections/app.style";
+
+import SEO from "../../components/seo";
+import Navigation from "../../sections/Navigation";
+import Footer from "../../sections/Footer";
+import Layout from "../../components/layout";
+import Conduct from "../../sections/Terms/index"
```
Comment (Tanuj22) [inline src/pages/terms/code-of-conduct.js:10]: `index` can be removed

File: src/sections/Terms/index.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import { Container, Row, Col } from "../../reusecore/Layout";
+import PageHeader from "../../reusecore/PageHeader"
+import SectionTitle from "../../reusecore/SectionTitle"
```
Comment (Tanuj22) [inline src/sections/Terms/index.js:5]: Remove this

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Good work @ashutosh1401. Some small changes - 

In the mobile view, the heading seems to be overlapping if multiple lines - 

Screenshot from 2020-10-12 13-39-43

File: src/sections/Terms/index.js (hunk 2 above)
Comment (ashutosh1401) [reply src/sections/Terms/index.js:5]: But Container Row and Col are being used by it provides a layout similar to other pages and sections

Comment (Tanuj22) [reply src/sections/Terms/index.js:5]: I meant only the `SectionTitile`

Comment (ashutosh1401) [reply src/sections/Terms/index.js:5]: OK

File: src/pages/terms/code-of-conduct.js (hunk 1 above)
Comment (ashutosh1401) [reply src/pages/terms/code-of-conduct.js]: Yes done

**Review by Tanuj22 — APPROVED** [review]
Thank you @ashutosh1401

**Pull request merged**